check.go

```diff
@@ -161,8 +170,7 @@ func init() {
 		return nil
 	})
 
-	// Scan fee-0.5 extension into Charges
-	path = "epp > response > extension > " + ExtFee + " chkData"
+	path = "epp > response > extension > " + ExtFee05 + " chkData"
```

Comment:
Can you comment to explain why this path is always `ExtFee05` regardless of the support determined above?